meetings/osf_oauth2_adapter/apps.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+import os
+
+class OsfOauth2AdapterConfig(AppConfig):
+    name = 'osf_oauth2_adapter'
+    # staging by default so people don't have to run OSF to use this.
+    osf_api_url = os.environ.get('OSF_API_URL', 'https://staging-api.osf.io').rstrip('/') + '/'
+    osf_accounts_url = os.environ.get('OSF_ACCOUNTS_URL', 'https://staging-accounts.osf.io').rstrip('/') + '/'
```

Comment:
I copied this code from osf [SHARE](https://github.com/CenterForOpenScience/SHARE/blob/develop/osf_oauth2_adapter/apps.py). The idea of it is to use staging by default but we can specify a different server with OSF_API_URL, and OSF_ACCOUNTS_URL.